exercise_02.py

```diff
@@ -0,0 +1,25 @@
+def reverse(number):
+    n_list = list(str(number))
+    number = ""
+    for i in range(1, len(n_list) + 1):
+        number += n_list[-i]
+
+    return int(number)
+
+
+def is_palindrome(number):
+    n_to_check = reverse(number)
+
+    return n_to_check == int(number)   
```

Comment:
`number` should be an `int` (assuming the user entered an integer) at this point. You should not convert it again.